cmd/installer/cli/join.go

```diff
@@ -618,3 +627,23 @@ func maybeEnableHA(ctx context.Context, kcli client.Client, flags JoinCmdFlags,
 		loading,
 	)
 }
+
+func joinDetermineBestNetworkInterface(joinAddress string) (string, error) {
+	// Try to use the interface that contains the ip address from the join command
```

Comment:
this assumes that the interface containing the first node's ip address is what's going to be used for traffic going to the first node, which might not always be the case no? specially in environments like GCP where routing is managed outside the VM.

Reply:
That is why it fails silently and falls back to the same detection behavior of the install command